x-pack/platform/plugins/shared/cloud/server/plugin.ts

```diff
@@ -200,6 +205,62 @@ export class CloudPlugin implements Plugin<CloudSetup, CloudStart> {
       projectType,
       orchestratorTarget,
     });
+    const basePath = core.http.basePath.serverBasePath;
+    core.http.resources.register(
+      {
+        path: '/app/cloud/onboarding',
+        validate: {
+          query: schema.maybe(
+            schema.object(
+              {
+                next: schema.maybe(schema.string()),
+                onboarding_token: schema.maybe(schema.string()),
+              },
+              { unknowns: 'ignore' }
+            )
+          ),
+        },
+        security: {
+          authz: {
+            requiredPrivileges: [],
```

Comment:
@azasypkin Let me know what to add here, I know it is failing the check type here.